data/events.json

```diff
@@ -0,0 +1,562 @@
+  },
+  {
+    "eventDbId": "6000000101",
+    "studyDbId": "6000000034",
+    "date": {
+      "discreteDates": ["2018-05-21"]
+    },
+    "eventDescription": "Ground-vehicle spray application of insecticide for treatment of Lepidopteran pests, especially Lesser Cornstalk Borer.  Belt SC (active ingredient Flubendiamide) applied at rate 0.029 Liters/Hectare (4 oz/acre), dilution per 100 gal = 20 oz.   ",
+    "eventParameters": [
+        {
+          "code": "CH_NAME",
+          "name": "chem_applic_name",
+          "unit": "text",
+          "value": "Belt SC"
+        },
+        {
+          "code": "CHCD",
+          "name": "chem_applic_material",
+          "unit": "text",
+          "value": "Flubendiamide"
+        },
+        {
+          "code": "CHAMT",
+          "name": "chem_applic_level",
+          "unit": "l/ha",
```

Comment:
The units for this ICASA variable are kg/ha, so this value could be converted using the chemical density.